concent_api/common/logging.py

```diff
@@ -258,6 +261,47 @@ def log(
         raise TypeError('Unexpected logging level')
 
 
+def log_received_force_payment(
+    logger: Logger,
+    force_payment_message: ForcePayment,
+):
+    log(logger, f"ForcePayment received with {len(force_payment_message.subtask_results_accepted_list)} nested SubtaskResultsAccepted.")
+    for subtask_results_accepted in force_payment_message.subtask_results_accepted_list:
+        log(logger, f"ForcePayment message contains SubtaskResultsAccepted with SUBTASK_ID: {subtask_results_accepted.subtask_id}.")
+
+
+def log_different_keys_addresses_and_wrong_requestor_signature(
+    logger: Logger,
+    subtask_results_accepted_list: List[SubtaskResultsAccepted],
+):
+    for subtask_results_accepted in subtask_results_accepted_list:
+        log(logger, f"Keys and addresses are not unique in {subtask_results_accepted.__class__.__name__} in "
+                    f"subtask_results_accepted_list and they are not signed by the same requestor", subtask_id=subtask_results_accepted.subtask_id)
+
+
+def log_duplicate_subtask_results_accepted(
```

Comment:
actually it is `subtask_id` being duplicated. Please change function name and the message logged. There is no need to print whole list, as the most important elements are printed when message is received.

Reply:
Done